cmd/root.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	//
+	// Or uncomment to load specific auth plugins
+	_ "k8s.io/client-go/plugin/pkg/client/auth/azure"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/openstack"
+)
+
+var RootCmd = &cobra.Command{
+	Use:   "pspmigrator",
+	Short: "pspmigrator is a tool to help migrate from PSP to PSA",
+}
+
+var (
+	clientset *kubernetes.Clientset
+	err       error
+	Namespace string
+)
+
+func init() {
+	initMutating()
+	RootCmd.AddCommand(MutatingCmd)
+	RootCmd.AddCommand(MigrateCmd)
+	var kubeconfig string
+
+	if home := homedir.HomeDir(); home != "" {
+		RootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "k",
+			filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
+	} else {
+		RootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "k", "", "absolute path to the kubeconfig file")
+	}
+
+	// use the current context in kubeconfig
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	config.WarningHandler = rest.NoWarnings{}
```

Comment:
because it would show a lot of warnings that PSP is being deprecated, but when someone is using this tool it's because they know it's because PSP is being deprecated. The output was full of warnings which wasn't great imo.

Reply:
example warning thrown if not surpressed:
```
W0721 21:17:20.823040   71459 warnings.go:70] policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+
```